README.md

```diff
@@ -1,2 +1,63 @@
 # funds-api
-Server APIs to interact with Toronto Mesh accounts on Stripe
+
+Server APIs to interact with Toronto Mesh accounts on [Stripe](https://stripe.com).
+
+This is a light-weight HTTP request handler to interact with the [Stripe Python Library](https://stripe.com/docs/libraries#python-library) using the account's [secret key](https://dashboard.stripe.com/account/apikeys), and should only be deployed on a secured server. The API server retrieves account information (e.g. transactions and balances) to render JSON responses containing information for public display. The `checkout` API is used to fulfill the server-side of a [Stripe checkout process](https://stripe.com/docs/checkout/tutorial).
+
+## Running the API server
+
+1. Make sure you have an updated version of Python
+
+1. Install the **Stripe Python Library**:
+
+    ```
+    $ sudo easy_install pip
+    $ sudo pip install --upgrade stripe
+    ```
+
+1. Set the environment variable `STRIPE_SECRET`:
+
+    ```
+    $ export STRIPE_SECRET=YOUR_STRIPE_SECRET_KEY
+    ```
+
+1. Clone this repository and run the API server in background:
+
+    ```
+    $ nohup python funds_api.py &
+    ```
+
+## Communicating to the running API server
+
+Assuming the website is served by an Apache HTTP server on the same host...
```

Comment:
I think for actual deployment we will use nginx. I was testing on my VM that uses Apache, so I wrote Apache instructions.